app/src/openxr/cpp/DeviceDelegateOpenXR.cpp

```diff
@@ -697,7 +697,8 @@ DeviceDelegateOpenXR::StartFrame(const FramePrediction aPrediction) {
     head.TranslateInPlace(vrb::Vector(-m.firstPose->position.x, -m.firstPose->position.y, -m.firstPose->position.z));
   #endif
 
-  if (m.renderMode == device::RenderMode::StandAlone) {
+  if (m.renderMode == device::RenderMode::StandAlone &&
+      (location.locationFlags & XR_SPACE_LOCATION_POSITION_VALID_BIT)) {
```

Comment:
I tested it with the OpenXR backend of the Oculus and it's causing some issues with the controller beams, so let's skip this for the moment and focus on the OVR fix.